examples/grid_example/public/app.tsx

```diff
@@ -123,7 +123,7 @@ export const GridExample = ({
         <EuiPageTemplate.Section
```

Comment:
For the mobile view, there's something wrong with calculating the height for the panels:

<img width="387" alt="Screenshot 2025-01-10 at 13 32 59" src="https://github.com/user-attachments/assets/06177775-0c1f-47a7-8567-b6d21155d8cb" />

checking: http://localhost:5601/app/dashboards#/view/edf84fe0-e1a0-11e7-b6d5-4dc382ef7f5b/